frontend/src/components/dashboard/aliases/AddressPickerModal.module.scss

```
@@ -29,21 +29,36 @@
       text-align: center;
       border-radius: $border-radius-md;
       background-color: $color-light-gray-10;
-      padding: $spacing-lg $spacing-md;
+      padding: $spacing-md $spacing-sm;
+
+      @media screen and #{$mq-sm} {
+        padding: $spacing-lg $spacing-md;
+      }
     }
 
     .warning {
       display: flex;
       align-items: center;
       gap: $spacing-lg;
-      padding: $spacing-lg $spacing-sm;
+      padding: $spacing-md $spacing-sm;
       border-bottom: 1px solid $color-light-gray-20;
       word-break: break-word;
 
+      @media screen and #{$mq-sm} {
+        padding: $spacing-lg $spacing-sm;
+      }
+
       .warning-icon {
+        // There is limited space on the mask creation modal on mobile.
+        // Hiding this icon until larger screens increases the available space for copy.
+        display: none;
         color: $color-pink-30;
         padding: $spacing-sm;
 
+        @media screen and #{$mq-sm} {
+          display: block;
```

Totally doesn't matter, but one thing I like to do here is to use `revert` to indicate that I'm undoing the rule above (`display: none`) rather than explicitly setting it to a particular value :)

```suggestion
          display: revert;
```